model-optimizer/extensions/back/SpecialNodesFinalization.py

```diff
@@ -123,7 +123,8 @@ def pattern():
         return dict(
```

Comment:
Update the transformation description with description why the `keep_output_port` attribute should not be equal to `True`